pkg/build/build.go

```diff
@@ -1227,6 +1228,20 @@ func (b *Build) buildWorkspaceConfig(ctx context.Context) *container.Config {
 		cfg.Environment[k] = v
 	}
 
+	for _, secret := range b.Secrets {
+		// Split secret into key value pair
+		parsedSecret := strings.Split(secret, "=")
+
+		// Append to environment if successfully split
+		if len(parsedSecret) == 2 {
+			k := parsedSecret[0]
+			v := parsedSecret[1]
+			cfg.Environment[k] = v
+		} else {
+			log.Errorf("could not parse secret: %s", parsedSecret)
```

Comment:
Since this variable likely contains a sensitive value (even if malformed), we probably don't want to log it

Reply:
That's a great point! I'll fail with the expected format instead